NineChronicles.Headless/GraphTypes/AddressQuery.cs

```diff
@@ -69,6 +72,36 @@ public AddressQuery()
                     }
                 ),
                 resolve: context => Addresses.GetRaiderListAddress(context.GetArgument<int>("raidId")));
+
+            Field<ListGraphType<AddressType>>(
+                name: "currencyMintersAddress",
+                description: "currency minters address.",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<CurrencyEnumType>>
+                    {
+                        Name = "currency",
+                        Description = "A currency type." +
+                                      "see also: https://github.com/planetarium/NineChronicles.Headless/blob/main/NineChronicles.Headless/GraphTypes/CurrencyEnumType.cs",
```

Comment:
If there is no space, it will be like `A currency type.see also: ...`

```suggestion
                        Description = "A currency type. " +
                                      "see also: https://github.com/planetarium/NineChronicles.Headless/blob/main/NineChronicles.Headless/GraphTypes/CurrencyEnumType.cs",
```